NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <add key="AspNetCore" value="https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json" />
+    <add key="AspNetCoreTools" value="https://dotnet.myget.org/F/aspnetcore-tools/api/v3/index.json" />
     <add key="AspNetCore-Patch" value="https://dotnet.myget.org/F/aspnetcore-rel-1-0-3/api/v3/index.json" />
```

Comment:
Which packages are required from this feed? Most packages on this feed should be on nuget.org now.